data.sql

```diff
@@ -2,7 +2,10 @@
 
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES
-	(1,'Agumon', TO_DATE('03/02/2020', 'MM/DD/YYYY'), 0, true, 10.23),
-	(2,'Gabumon', TO_DATE('11/15/2020', 'MM/DD/YYYY'), 2, true, 8),
-	(3,'Pikachu', TO_DATE('01/07/2021', 'MM/DD/YYYY'), 1, false, 15.04),
-	(4,'Devimon', TO_DATE('05/12/2017', 'MM/DD/YYYY'), 5, true, 11);
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you don't remove the previous queries from your project because these queries are related to your next milestone queries so please don't delete them keep all the queries in your project 👍